src/test/common/clara/test_testing_utils.cljc

```diff
@@ -0,0 +1,56 @@
+#?(:clj
```

Comment:
It is somewhat non-idiomatic to use a read conditional at this scope.

It is more common to see them used just in the places where the forms differ. Here there is duplication and even a chance the `ns` name could be changed, etc. 

It's not a huge issue though.